internal/ocm/resource/cluster.go

```diff
@@ -1,13 +1,18 @@
 package resource
 
 import (
-	"context"
 	"errors"
+	"fmt"
+	"regexp"
 
 	"github.com/openshift-online/ocm-common/pkg/cluster/validations"
 	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
 )
 
+var kmsArnRE = regexp.MustCompile(
```

Comment:
Can we move it into the common repo?

Reply:
Opened a jira ticket [OCM-3155](https://issues.redhat.com/browse/OCM-3155)
This will be added in different PR